prow/plugins/label/label.go

```diff
@@ -126,6 +142,7 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	labelMatches := labelRegex.FindAllStringSubmatch(ae.body, -1)
 	removeLabelMatches := removeLabelRegex.FindAllStringSubmatch(ae.body, -1)
 	sigMatches := sigMatcher.FindAllStringSubmatch(ae.body, -1)
+	isMember, _ := gc.IsMember(ae.org, ae.login)
```

Comment:
can you only make this call if there are sigMatches? *most* comments won't mention a sig at all, so wasting an API call checking for membership is pointless.